lib/address_geocoder.rb

```diff
@@ -56,7 +56,7 @@ def call_google
         if attempts <= 5
           retry
         else
-          raise ConnectionError, 'Could not connect to GoogleAPI'
+          raise SystemCallError, 'Could not connect to GoogleAPI'
```

Comment:
The first one seems to be mainly concerned with Net::HTTP, don't understand how it applies unless we want to rescue different errors in different ways. Creating our own sounds cool cause we could call it whatever we wanted. I think if we also did raise without the error-name it would throw the original error.